mappings/net/minecraft/world/spawner/CatSpawner.mapping

```diff
@@ -0,0 +1,27 @@
+CLASS net/minecraft/class_4274 net/minecraft/world/spawner/CatSpawner
+	COMMENT A spawner for cats in villages and swamp huts.
+	COMMENT
+	COMMENT @implNote Cats in swamp huts are also spawned in
+	COMMENT {@link net.minecraft.world.gen.chunk.ChunkGeneratorSettings#getEntitySpawnList}.
+	FIELD field_19171 cooldown I
+	FIELD field_30601 SPAWN_INTERVAL I
+	METHOD method_20260 spawnInSwampHut (Lnet/minecraft/class_3218;Lnet/minecraft/class_2338;)I
+		COMMENT Tries to spawn cats in swamp huts.
+		COMMENT
+		COMMENT @return the number of cats spawned
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_20262 spawn (Lnet/minecraft/class_2338;Lnet/minecraft/class_3218;)I
+		COMMENT Spawns a cat.
+		COMMENT
+		COMMENT @return the number of cats spawned
```

Comment:
"*a* cat" but "number of cats spawned"? If it can spawn multiple, "Spawns cats" might be better.

Reply:
It cannot spawn multiple cats in vanilla. However, I didn't want to document this as "returns 1 when successful and 0 when failed", because these are not just simple booleans. (If the code were to be modified to spawn multiple cats, it should return the number of cats spawned.)